.github/workflows/lint.yml

```diff
@@ -35,3 +35,9 @@ jobs:
         run: npm install -g prettier
       - name: run prettier
         run: prettier -c .
+      - name: run prettier
+        if: ${{ failure() }}
+        run: prettier -w .
+      - name: show prettier diff
+        if: ${{ failure() }}
+        run: git diff
```

Comment:
Let me know if this is okay in this PR

Reply:
Lets remove it as part of landing it; I want to annotate output from prettier.

Reply:
Done, removed the commit